src/site/content/en/fast/codelab-text-compression-brotli/index.md

```diff
@@ -193,7 +193,7 @@ Begin by adding it as a `devDependency` in `package.json`:
 Like any other webpack plugin, import it in the configurations file,
 `webpack.config.js`:
 
-```js/4
+```js/3
```

Comment:
maybe remove the closing curly on line 201? It looks kind of like a typo in the example.